next/components/sections/WithoutMenuSection.tsx

```diff
@@ -1,20 +1,21 @@
 import { HomepageHeaderFragment } from '@bratislava/strapi-sdk-homepage'
-import { BAStickyMenu, MenuMainItem, SectionContainer } from '@bratislava/ui-bratislava'
+import { MenuMainItem, SectionContainer } from '@bratislava/ui-bratislava'
 import HomepageMenu from '@bratislava/ui-bratislava/HomepageMenu/HomepageMenu'
 import cx from 'classnames'
-import { useState } from 'react'
+import { MutableRefObject, useState } from 'react'
 
 import HomePageSearch from '../molecules/HomePageSearch'
-import { STICKY_MENU_STATE, useWelcomeSection } from './useWelcomeSection'
+import { STICKY_MENU_STATE, useWithoutMenuSection } from './useWithoutMenuSection'
 
 interface Props {
   homepageHeader: HomepageHeaderFragment | null | undefined
   mainMenuItems: MenuMainItem[]
+  elementRef: MutableRefObject<HTMLDivElement | null>
 }
 
-export const WelcomeSection = ({ mainMenuItems, homepageHeader }: Props) => {
+export const WithoutMenuSection = ({ mainMenuItems, homepageHeader, elementRef }: Props) => {
```

Comment:
What was wrong with `WelcomeSection` name? The section is clearly the first thing you see on the web, it even says "welcome" above the searchbar :D so it's suitable to call it Welcome section.
It really took me a while to understand what you meant by "WithoutMenuSection" because it acctually contains menu in it :D.
Moreover, the fact that the sticky menu is not shown initially just on homepage can be changed.